docs/general/administration/memory-dumping.md

```diff
@@ -0,0 +1,101 @@
+---
+uid: server-memorydumps
+title: Server Memory Dumps
+---
+
+# Memory Dumping of the Jellyfin server
+
+To troubleshoot a jellyfin server that keeps allocating system memory it is nessesary to dump the memory in a format that the developers then can later analyse.  
+
+To dump the memory we use the dotMemory tools from Jetbrains.
+
+## Linux Barebones
+
+First we need to install the latest dotMemory commandline tooling.
+To do this we pull the nuget package and extract it to a folder named `dotMemoryClt` and then set the permissions to be executable:
+
+```sh
+apt-get update -y && apt-get install -y wget && \
+wget -O dotMemoryclt.zip https://www.nuget.org/api/v2/package/JetBrains.dotMemory.Console.linux-x64/2024.2.2 && \
+apt-get install -y unzip && \
+unzip dotMemoryclt.zip -d ./dotMemoryclt && \
+chmod +x -R dotMemoryclt/*
+```
+
+afterwards we need to figure out the process ID of the jellyfin server via
+
+```sh
+ps aux
+```
+
+If the ps command is not available in your Linux distribution, install it with:
+
+```sh
+apt-get update && apt-get install procps
+```
+
+this will show us a number of processes like this:
+
+```cmd
+# ps aux
+USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
+root         1  1.8  7.4 276905428 2304952 ?   Ssl  Sep04  17:23 /jellyfin/jellyfin
+root       914  0.0  0.0   2480   580 pts/0    Ss   05:33   0:00 sh
+root      2171  0.0  0.0   6756  2940 pts/0    R+   12:55   0:00 ps aux
+```
+
+Now we note the process ID (PID) of the jellyfin server. That is the process that has the `/jellyfin/jellyfin` command. The left part of the path might differ if you have installed jellyfin in a different directory but the most right part of the path should always be `/jellyfin`.
+
+Then we run the memory profiler via this command with the `{PID}` replaced by the pid we noted earlier (In this case it would be 1):
```

Comment:
```suggestion
Then we run the memory profiler via this command with the `{PID}` replaced by the PID we noted earlier (In this case it would be 1):
```